### PR TITLE
Improve package json

### PR DESCRIPTION
**Pull request by shomix**
- add description
- add keywords
- add author
- remove duplicate license

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "sdk",
   "version": "0.0.0",
+  "author": "Spheron Team",
```
Comment (rekpero) [inline package.json:4]: put a description here too, use the description of the repo for this

File: packages/storage/package.json [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
   "name": "@spheron/storage",
   "version": "1.0.0",
+  "description": "Typescript library for uploading files or directory to  IPFS, Filecoin or Arweave via Spheron",
+  "keywords": [
+    "Spheron",
+    "Network",
+    "Hosting",
+    "Storage",
+    "Typescript",
+    "Badass"
```
Comment (rekpero) [inline packages/storage/package.json:11]:
```suggestion
    "Badass", "IPFS", "Filecoin", "Arweave", "SDK", "Upload"
```

File: packages/storage/package.json [hunk 3]
```diff
@@ -1,6 +1,16 @@
 {
   "name": "@spheron/storage",
   "version": "1.0.0",
+  "description": "Typescript library for uploading files or directory to  IPFS, Filecoin or Arweave via Spheron",
+  "keywords": [
+    "Spheron",
+    "Network",
+    "Hosting",
```
Comment (rekpero) [inline packages/storage/package.json:8]: remove this

**Review by rekpero — CHANGES_REQUESTED** [review]
small suggestion added, please check @shomix

File: package.json (hunk 1 above)
Comment (shomix) [reply package.json:4]: added

**Review by rekpero — APPROVED** [review]

**Pull request merged**